tests/lisa/test_trace.py

```diff
@@ -245,6 +246,37 @@ def test_sched_load_avg_signals(self):
         """Test parsing sched_load_avg_task events from EAS1.2"""
         self._test_tasks_dfs('sched_load_avg')
 
+    def test_getPeripheralClockInfo(self):
+        """
+        TestTrace: getPeripheralClockInfo() returns proper effective rate info.
+        """
+        self.events = [
+            'clock_set_rate',
+            'clock_disable',
+            'clock_enable'
+        ]
+        trace = self.make_trace("""
+          <idle>-0 [002] 380330000000: clock_enable: bus_clk state=1 cpu_id=2
+          <idle>-0 [002] 380331000000: clock_set_rate: bus_clk state=750000000 cpu_id=2
+          <idle>-0 [000] 380332000000: clock_disable: bus_clk state=0 cpu_id=0
+          <idle>-0 [000] 380333000000: clock_enable: bus_clk state=1 cpu_id=0
+          <idle>-0 [002] 380334000000: clock_set_rate: bus_clk state=100000000 cpu_id=2
+          <idle>-0 [000] 380335000000: clock_disable: bus_clk state=0 cpu_id=0
+          <idle>-0 [004] 380339000000: cpu_idle:             state=1 cpu_id=4
+        """)
+        df = trace.getPeripheralClockEffectiveRate(clk_name='bus_clk')
+        print(df)
```

Comment:
Eh, could have used that trace to test my previous comments ;_;
Anyway, tests are cool, so thanks for adding one. I'd just like the print to go please.